pages/agent/v3/tracing.md.erb

```diff
@@ -0,0 +1,24 @@
+# Tracing in the Buildkite Agent
+
+Distributed tracing tools like Datadog APM or OpenTelemetry tracing allow you to gain more insight in the the performance of your CI runs - what's fast, what's slow, what could be optimized, and more importantly, how these things are changing over time.
+
+The Buildkite Agent currently supports the two tracing backends listed above, Datadog APM (via OpenTracing) and OpenTelemetry. This doc will guide your through setting up tracing using either of these backends.
+
+{:toc}
+
+## Using Datadog APM
+
+To use the Datadog APM integration, start the Buildkite agent with the `--tracing-backend datadog` option. This will enable Datadog APM tracing, and send the traces to a Datadog Agent at `localhost:8126` by default. If your Datadog Agent is located at another host, the Buildkite Agent will respect the `DD_AGENT_HOST` and `DD_AGENT_TRACE_PORT` environment variables defined by [`dd-trace-go`](https://docs.datadoghq.com/tracing/setup_overview/setup/go/?tab=containers). Note that there will need to be a Datadog agent present at the above address to ingest these traces.
+
+Once this is done, and the will start sending tracing information to Datadog. You can observe traces as they come in in the APM > Traces screen in your Datadog instance.
```

Comment:
> and the will start
looks like a bit is missing here

Reply:
@karensawrey it's fixed, but doesn't show in the discussion view for some reason - see https://github.com/buildkite/docs/pull/1512/files#diff-59e3d95ef9811d2b62c22ebe56c9865456b102166ac3353c119ff3a8f47c0651R13